src/style.scss

```diff
@@ -161,7 +186,23 @@ input::-ms-clear {
 
 
 %activeLabel {
-  font-size: var(--auro-text-body-size-sm);
+  left: 0;
+
   padding: 0;
   padding-top: var(--auro-size-xs);
+
+  font-size: var(--auro-text-body-size-sm);
+}
+
+.creditCard {
+  padding-left: 30px;
+}
+
+.creditCard-icon {
```

Comment:
Positioning needs to be adjusted to support when helptext is added or there is an error state.